huxley/www/static/js/huxley/components/RegistrationView.js

```diff
@@ -89,6 +89,14 @@ var RegistrationView = React.createClass({
     };
   },
 
+  componentWillMount: function() {
+    if (!conference.registration_open) {
```

Comment:
Is there a reason why we're putting this logic here instead of on the Register button in the LoginView. We should condition on `conference.registration_open` such that if it's true the `href` on the Register button is `register/open` and if it's false the href is `register/closed`. We should do something similar for waitlisting schools by conditioning on `conference.registration_waitlist` such that if its true in the onSuccess function for the ajax call in RegistrationView it transitions to `register/waitlist` or `register/success` otherwise.